pyproject.toml

```diff
@@ -1,7 +1,41 @@
+[build-system]
+requires = [
+    "setuptools >= 56.0",
+    "wheel >= 0.29.0",
+    "versioningit ~= 1.1.1",
+]
+build-backend = 'setuptools.build_meta'
+
+[tool.pytest.ini_options]
+junit_family = "legacy"
+addopts = "--strict-markers"
+
 [tool.mypy]
 ignore_missing_imports = true
 show_error_codes = true
 enable_error_code = "ignore-without-code"
 warn_unused_ignores = true
 warn_unused_configs = true
 warn_redundant_casts = true
+
+[[tool.mypy.overrides]]
+module = [
+    "broadbean.tests.*",
+]
+disallow_untyped_defs = false
```

Comment:
This is strictly not needed since we never set it to true (broadbean is not fully typed yet)